### PR TITLE
Neutron RBAC Blog entry

### PR DESCRIPTION
**Pull request by busterswt**
author(s) mini bio, soft capped to 75 words:

James Denton is a Principal Architect for the Rackspace Private Cloud team, and is the author of 'Learning OpenStack Networking (Neutron) - Second Edition' available on Amazon and packtpub.com.

author(s) social media contact info: @jimmdenton

accept or decline to allow redaction, editing or structure changes by technical writer staff: yes

desired date of publishing: ASAP

### CODE REVIEW

File: _posts/2015-12-03-A-First-Look-at-RBAC-in-the-Liberty-Release-of-Neutron.md [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+comments: true
+author: James Denton
+published: true
+categories:
+    - neutron
+    - rpc
+    - openstack
+---
+
+Over the last couple of years, we've seen OpenStack deployments shift from a public cloud model where no one is trusted to a private cloud model where collaboration and shared resources between projects is required. As enterprises adopt OpenStack and integrate it into their infrastructure, new use cases continue to multiply and existing limitations in APIs and data models have been brought to the forefront. One of the more exciting features to come out of Neutron development in the Liberty cycle that addresses a shortcoming is a framework for Role Based Access Control (RBAC).
```
Comment (catlook) [inline _posts/2015-12-03-A-First-Look-at-RBAC-in-the-Liberty-Release-of-Neutron.md]: ... model, where no one is trusted, to a  ...
... cloud model, where collaboration ...
... to multiply, and existing limitations ...

File: _posts/2015-12-03-A-First-Look-at-RBAC-in-the-Liberty-Release-of-Neutron.md [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+    - neutron
+    - rpc
+    - openstack
+---
+
+Over the last couple of years, we've seen OpenStack deployments shift from a public cloud model where no one is trusted to a private cloud model where collaboration and shared resources between projects is required. As enterprises adopt OpenStack and integrate it into their infrastructure, new use cases continue to multiply and existing limitations in APIs and data models have been brought to the forefront. One of the more exciting features to come out of Neutron development in the Liberty cycle that addresses a shortcoming is a framework for Role Based Access Control (RBAC). 
+
+<!-- more -->
+
+RBAC aims to solve the inability to share certain Neutron resources with a subset of projects or tenants. Neutron has supported shared resources in the past, but up until now it's been all-or-nothing. If a network or other resource is marked as `shared`, it is shared with *all* tenants with no ability to specify otherwise. RBAC policies will allow administrators and users to share resources with one or more tenants using a granular, rather than a shotgun, approach.
```
Comment (catlook) [inline _posts/2015-12-03-A-First-Look-at-RBAC-in-the-Liberty-Release-of-Neutron.md]: ... but, until now, it's been  ...

File: _posts/2015-12-03-A-First-Look-at-RBAC-in-the-Liberty-Release-of-Neutron.md [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+ERROR (BadRequest): Network c4471d6a-ab58-4ea3-971d-39104a5ad509 could not be found. (HTTP 400) (Request-ID: req-ce3fbce5-afec-41e1-8004-a8abddfad5eb)
+```
+
+# Caveats, warranties, etc.
+
+Neutron's RBAC functionality is undergoing active development, and you're likely to see ~~regressions~~ improvements already in the `MASTER` branch and upcoming Mitaka release. What I've demonstrated here is functionality that a lot of enterprises are asking for, and hopefully just a glimpse of what should be possible once the feature matures. 
+
+If you'd like to learn more about Neutron's RBAC functionality, take a look at the specs and blueprints found on [specs.openstack.org](https://specs.openstack.org/openstack/neutron-specs/specs/liberty/rbac-networks.html). As always, feel free to reach out to me on Twitter @jimmdenton with any questions!
+
+
```
Comment (catlook) [inline _posts/2015-12-03-A-First-Look-at-RBAC-in-the-Liberty-Release-of-Neutron.md]: [ ed. note: Nice article and demo!!  Editing done -  Cat ]

**Comment by busterswt** [comment]
I'm good with the changes :) Feel free to publish today if you'd like!

**Comment by catlook** [comment]
James,

Go ahead and merge the changes into the blog master, and we'll schedule the publish for tomorrow :)

-- Cat Lookabaugh, Sr. Information Developer
Rackspace Hosting - the #1 Managed Cloud Company
DevDocs on Developer Experience Team
Office: (210) 312-8193   Cell: (210) 364-1060
cat.lookabaugh@rackspace.com

We should not write so that it is possible for the reader to understand us, but so that it is impossible for him to misunderstand us. - Quintilian, rhetorician (c. 35-100)

Harmony + Learner + Responsibility + Context + Intellection + Arranger + Consistency + Relator + Input + Analytical

File: _posts/2015-12-03-A-First-Look-at-RBAC-in-the-Liberty-Release-of-Neutron.md [hunk 4]
```diff
@@ -0,0 +1,324 @@
+---
+layout: post
+title: "A First Look at RBAC in the Liberty Release of Neutron"
+date: 2015-12-03 08:00:00
+comments: true
```
Comment (everett-toews) [inline _posts/2015-12-03-A-First-Look-at-RBAC-in-the-Liberty-Release-of-Neutron.md]: In your PR comment you didn't mention whether or not you're willing to answer comments about your post.

If not, please set `comments: false`.

If so, be sure to subscribe to your post comments by clicking on the Subscribe link at the very bottom of the post once it's published.

**Pull request merged**